### DISCUSSION TITLE
Text Strike

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1295] [Proposals]
### Goal

Habilitar no Figma a formatação text strike para que os Designers possam aplicar nas interfaces.

### Context

Em alguns fluxos do Novo App, usamos a formatação text strike nas interfaces. Em código é possível aplicar essa formatação, mas no Figma ainda não.

Link Playroom: https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8BnAfEgKjAHgFwAJIAbCAJwF4AdEGtFCAWxgL1zyQHpt8NP0qAOyEgANCDwALGMxQIA2iABqASwBuEEAF1xAdxVQpc%2BPIDMANgAMWgL5A

### Impact

Atualmente, os Designers, estão precisando dar "detach" no text-preset para aplicar a formatação text strike.

Queremos deixar essa formatação de texto como uma opção para que a pessoa Designer faça a aplicação da maneira correta.

### Discussion Questions

É possível fazer essa evolução no Mística?

### Do you have a visual proposal?

https://www.figma.com/file/vABLnMzf2o1m6zAvtKcnFN/Text-Strike?type=design&node-id=0%3A1&t=nYxEk1dzeYVQV27r-1

**Comment by aweell** [comment]
Hello @annefranca,

This is a limitation of Figma. Even though is possible to create text styles with strike-trough, we would need to duplicate all the current to allow this configuration. This is a level of maintenance and complexity that we want to avoid.

We hope that upcoming Figma updates will bring the possibility to change the text decoration without detaching the style.

https://forum.figma.com/t/allow-modifying-decoration-to-the-text-styles-already-defined/807

Until then, we believe that the best way to achieve strike-trough in texts is still the "detach" approach.
